### PR TITLE
Impl 157

### LINKED ISSUE #157: Provide an option to be able to show the full width content above the and below the map
screenshot_10

### PR DESCRIPTION
**Pull request by Huyeng**
Fixes #157

### CODE REVIEW

File: post-types/profile-pages.php [hunk 1]
```diff
@@ -342,7 +339,7 @@ public function profiles_page_settings_box($post = false)
               <table class="form-table form-table-localization profiles_page_settings_box">
                 <tbody>
                  <tr>
-                  <th><label for="_total_number_by_attribute_name_localization"><?php _e('Show Total Numbers of Columns, separated by line breaks ('.odm_language_manager()->get_the_language_by_site().')', 'wp-odm_profile_pages'); ?></label></th>
+                  <th><label for="total_number_by_attribute_name_localization"><?php _e('Show Total Numbers of Columns, separated by line breaks ('.odm_language_manager()->get_the_language_by_site().')', 'wp-odm_profile_pages'); ?></label></th>
                   <td>
             			<textarea name="_total_number_by_attribute_name_localization" style="width:100%;height: 80px;"placeholder="column_1"><?php echo $total_number_by_attribute_name_localization; ?></textarea>
```
Comment (acorbi) [inline post-types/profile-pages.php]: @Huyeng you need to add ```ìd="total_number_by_attribute_name_localization"``` to the <textarea> so the <label> is  related to it.

File: post-types/profile-pages.php [hunk 2]
```diff
@@ -370,14 +367,14 @@ public function profiles_page_settings_box($post = false)
...
+            				<input id="link_to_detail_column_localization" type="text" name="_link_to_detail_column_localization" size="40" placeholder="name,company,developer,block" value="<?php echo $link_to_detail_column_localization? $link_to_detail_column_localization : "name"; ?>" />
                   <p class="description"><?php _e('Please add the ids of the columns that will feature a link to the entry\'s detail page. Format: Comma-separated values. <br/>eg. name,company,developer,block', 'wp-odm_profile_pages'); ?></p>
           		  </td>
           		 </tr>
                <tr>
-    					  <th><label for="_link_to_detail_page_localization"><?php _e('Select the column id to use for Detail Link ('.odm_language_manager()->get_the_language_by_site().')', 'wp-odm_profile_pages'); ?></label></th>
+    					  <th><label for="link_to_detail_page_localization"><?php _e('Select the column id to use for Detail Link ('.odm_language_manager()->get_the_language_by_site().')', 'wp-odm_profile_pages'); ?></label></th>
     					  <td>
                   <select class="link_to_detail_page_localization" name="_link_to_detail_page_localization">
```
Comment (acorbi) [inline post-types/profile-pages.php]: @Huyeng add ```id="link_to_detail_page_localization" to this <select>

File: post-types/profile-pages.php [hunk 3]
```diff
@@ -432,12 +430,17 @@ public function full_width_middle_content_box($post = false)
...
-                ?></p>
+                <td>
+                <div style="float:left; margin-bottom:1em">
+                    <label for="_full_width_middle_content"><?php _e('Full width content (English)', 'wp-odm_profile_pages');?></label>
+                </div>
+                <div style="float:right; margin-bottom:1em">
+                    <input type="checkbox" name="_full_width_content_position" id="full_width_content_position" value="1" <?php checked(1, $show_above_map);?>>
+                    <label for="full_width_content_position"><?php _e('Show above the map', 'odm'); ?></label>
+                </div>
+                  <textarea name="_full_width_middle_content" style="width:100%;height: 50px;" placeholder=""><?php echo $full_width_middle_content; ?></textarea>
```
Comment (acorbi) [inline post-types/profile-pages.php]: @Huyeng here as well id="full_width_content_position"

File: post-types/profile-pages.php [hunk 4]
```diff
@@ -448,7 +451,7 @@ public function full_width_middle_content_box($post = false)
             <table class="form-table form-table-localization middle_content_box">
               <tbody>
                 <tr>
-                <td><label for="_full_width_middle_content_localization"><?php _e('Full width content (('.odm_language_manager()->get_the_language_by_site().')', 'wp-odm_profile_pages');
+                <td><label for="_full_width_middle_content_localization"><?php _e('Full width content ('.odm_language_manager()->get_the_language_by_site().')', 'wp-odm_profile_pages');
                 ?></label>
                 </br>
                 <textarea name="_full_width_middle_content_localization" style="width:100%;height: 50px;" placeholder=""><?php echo $full_width_middle_content_localization; ?></textarea>
```
Comment (acorbi) [inline post-types/profile-pages.php]: @Huyeng here as well ```id="_full_width_middle_content_localization"```

**Comment by acorbi** [comment]
@Huyeng and do not forget to increase version number

File: post-types/profile-pages.php (hunk 2 above)
Comment (Huyeng) [inline post-types/profile-pages.php]: Oh yes. I didn't check if the textarea has ID or not.

**Pull request merged**